crates/ethereum/consensus/src/lib.rs

```diff
@@ -53,6 +53,15 @@ impl<ChainSpec: EthChainSpec + EthereumHardforks> EthBeaconConsensus<ChainSpec>
         header: &SealedHeader<H>,
         parent: &SealedHeader<H>,
     ) -> Result<(), ConsensusError> {
+        // TODO: Maybe put this check in another place, it is not really
+        // TODO: related to the parent gas limit
```

Comment:
TODO: Just due to unfamiliarity with where all of the checks are

Reply:
I believe this belongs here:

https://github.com/paradigmxyz/reth/blob/main/crates/consensus/common/src/validation.rs#L13-L15

the way these checks are structured that we have fn that should check related values, and checks that can be applied on the header directly

Reply:
Yep makes sense